wai-extra/Network/Wai/Middleware/RequestLogger.hs

```diff
@@ -9,11 +10,13 @@ module Network.Wai.Middleware.RequestLogger
       -- * Create more versions
     , mkRequestLogger
     , RequestLoggerSettings
+    , def
```

Comment:
I noticed this is exported in the `Gzip` module so far. Exporting it here could break some imports that import both `Gzip` and `RequestLogger` at the same time. Importing from `Gzip` and using it for `RequestLogger` seems... wrong, though :D

Not sure, however, to deal best with this?

Reply:
Don't use the `Default` typeclass. Instead, expose a new identifier such as `defaultApacheSettings`.